docs/http/traffic-policy/index.mdx

```diff
@@ -86,15 +86,15 @@ See [actions](./actions) for all available actions.
 
 ### Configuration
 
-| Parameter       | Description                                                                                                            |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------- |
-| **inbound**     | A list of policy rules that will be applied to inbound traffic in the order specified.                                 |
-| **outbound**    | A list of policy rules that will be applied to outbound traffic in the order specified.                                |
-| **name**        | Policy rules can optionally be given a name for convenience.                                                           |
-| **expressions** | A list of CEL expressions that filter which traffic a policy rule will apply to.                                       |
-| **actions**     | A list of actions that will execute sequentially if the associated policy rule's expressions all match on the traffic. |
-| **type**        | The type of action.                                                                                                    |
-| **config**      | The configuration details of how an action should execute. Each action has its own configuration structure.            |
+| Parameter            | Description                                                                                                            |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| **on_http_request**  | A list of policy rules that will be applied to inbound http traffic in the order specified.                            |
+| **on_http_response** | A list of policy rules that will be applied to outbound http traffic in the order specified.                           |
```

Comment:
Hmm idk how this table got this way to begin with but these seems confusing that we have the phases defined in the same table as the rule properties?

Reply:
we can remove it from the table if we want, i don't mind! @nijikokun thoughts?